src/components/augmentedReality/AugmentedRealityView.js

```diff
@@ -76,41 +71,83 @@ export const AugmentedRealityView = ({ sceneNavigator }) => {
   );
 };
 
+// eslint-disable-next-line complexity
 const ViroSoundAnd3DObject = (item) => {
   const {
     isObjectLoading,
     isStartAnimationAndSound,
     object,
-    position,
-    rotation,
-    scale,
     setIsObjectLoading,
     setIsStartAnimationAndSound
   } = item;
 
+  if (object?.mp4) {
+    // if the `chromaKeyFilteredVideo` prop is undefined, the default colour of GreenScreen is defined
+    ViroMaterials.createMaterials({
+      chromaKeyFilteredVideo: {
+        chromaKeyFilteringColor: object?.mp4?.chromaKeyFilteredVideo || '#00FF00'
+      }
+    });
+  }
+
   return (
     <>
-      {!!object.mp3 && !isObjectLoading && (
-        <ViroSound
-          source={{ uri: object.mp3 }}
-          paused={!isStartAnimationAndSound}
-          onFinish={() => setIsStartAnimationAndSound(false)}
-        />
+      {isObjectLoading && (
```

Comment:
should be `!isObjectLoading` as before, right?

Reply:
you also wrote that in the commit message: `added a main control with `!isObjectLoading` because...` :)